tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -309,7 +309,7 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
```

Comment:
how about adding here just simple to explain port, what it's for and that people can change it if they want to something else "(`-p 8888:8888` used to share internal port to the host machine, in this case to ensure to ensure Jupiter notebook connection)".